### ISSUE TITLE
Errors in lake serve are printed on stdout

### ISSUE THREAD

**Issue by Vtec234** [leanprover/lake#72]
When `lake serve` encounters errors `lakefile.lean`, it prints them out on `stdout` rather than `stderr`. For example
```bash
nawrocki@gentzen:~/npm-widget$ lake serve 1>/dev/null
^C
nawrocki@gentzen:~/npm-widget$ lake serve 2>/dev/null
./lakefile.lean:7:40: warning: unused variable `srcFile`
^C
```
With `lake serve`, `stdout` should only ever speak LSP and errors should be directed to `stderr`. We could then detect them in `vscode-lean4` and show them to the user. See also [Zulip discussion](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/VSCode.20.22Waiting.20for.20lean.20server.20to.20start.2E.2E.2E.22/near/282604102).

**Comment by tydeu** [comment]
I believe this has already been fixed by #71. Thus, it should be in fixed in the next merge of Lake upstream (which I am on schedule to PR by the end of the week).

**Comment by Vtec234** [comment]
Oh, awesome, thanks! I will wait for the release.
